packages/AlgernonTests.package/ALGTestActionVerbs.class/instance/testSaveAndQuit.st

```diff
@@ -0,0 +1,5 @@
+testing
+testSaveAndQuit
+	self runQuery: 'saveandquit'.
+	self assert: (self items first algerType typeName = 'Quit').
+	self assert: (self items first name = 'saveandquit')
```

Comment:
You may want to use `nameOfFirstResult` here (and create something similar for typeName).